cdap-ui/app/cdap/components/Overview/Tabs/SchemaTab/index.js

```diff
@@ -37,9 +36,14 @@ export default class SchemaTab extends Component {
 
   componentWillMount() {
     if (!this.props.entity.schema) { return; }
-
-    let fields = getParsedSchema(this.props.entity.schema);
```

Comment:
I noticed that we have code identical to these removed lines in the `componentWillReceiveProps`. Would we have to change them as well? Also not exactly sure why the new code changes work.